packages-ts/starknet-gauntlet-cli/src/index.ts

```diff
@@ -48,6 +48,7 @@ const registerExecuteCommand = <UI, CI>(
         billingAccessController: process.env.BILLING_ACCESS_CONTROLLER,
         link: process.env.LINK,
         secret: flags.secret || process.env.SECRET,
+        withLedger: !!flags.withLedger,
```

Comment:
Let's include `ledgerPath` here. If undefined, it will fall back to the default.

Reply:
Also, `withLedger` could come defined by `process.env.WITH_LEDGER`